crates/uv/tests/it/tool_install.rs

```diff
@@ -3922,6 +3962,7 @@ fn tool_install_default_credentials() -> Result<()> {
         [tool.options]
         index = [{ url = "https://pypi-proxy.fly.dev/basic-auth/simple", explicit = false, default = true, format = "simple", authenticate = "always" }]
         exclude-newer = "2025-01-18T00:00:00Z"
+        compile-bytecode-timeout = { secs = 60, nanos = 0 }
```

Comment:
Hm, I don't think we should be persisting the default value here. If it wasn't explicitly configured, then it shouldn't be persisted for subsequent operations.